pkg/azuredisk/azuredisk.go

```diff
@@ -138,3 +140,17 @@ func (d *Driver) checkDiskExists(ctx context.Context, diskURI string) error {
 
 	return nil
 }
+
+func (d *Driver) validateDiskURI(diskURI string) error {
```

Comment:
would you put this `validateDiskURI` to `azuredisk.go` and add a unit test? Thanks. And 
 - rename this function to `isValidDiskURI`
 - this function don't need to be a member function of `(d *Driver)`, you could refer to `isManagedDisk`

Reply:
Okay, I have made the changes and added the tests.